region/lowernorfair/west.json

```diff
@@ -169,15 +178,51 @@
                   "name": "Acid Dive",
                   "notable": false,
                   "requires": [
-                    "canSuitlessLavaDive",
+                    "h_canNavigateHeatRooms",
+                    {"or": [
+                      "Gravity",
```

Comment:
This would still be suitless lava dive, even in gravity is owned

Reply:
The odd looking part is that you jump on a pretty high ledge.  With HiJump you dive right to the bottom.  It probably still is suitlessLavaDive as its described.